packages/smart-contracts/test/contracts/EthConversionProxy.test.ts

```diff
@@ -95,10 +96,9 @@ describe('contract: EthConversionProxy', () => {
           .sub(feeOldBalance)
           .toString();
 
-        const gasPrice = (await provider.getFeeData()).gasPrice || 0;
         // Check balance changes
-        expect(fromNewBalance.toString()).to.equals(
-            fromOldBalance.sub(conversionToPay.result).sub(conversionFees.result).sub(receipt.gasUsed.mul(gasPrice)).toString(),
+        expect(Number(fromNewBalance)).to.be.lessThan(
```

Comment:
If `fromNewBalance` is a BigNumber, it's a bit better to use `fromNewBalance.lte(fromOldBalance.sub(...))`